src/operator/zip.ts

```diff
@@ -61,6 +61,30 @@ export function zipStatic<R>(...observables: Array<ObservableInput<any> | ((...v
 /* tslint:enable:max-line-length */
 
 /**
+ * Combines multiple Observables to create an Observable whose values are calculated from the values, in order, of each 
+ * of its input Observables.
+ *
+ * If the latest parameter is a function, this function is used to compute the created value from the input values. 
+ * Otherwise, an array of the input values is returned.
+ *
+ * @example <caption>Combine age, name and gender from different sources</caption>
+ *
+ * let Ages = Observable.of<number>(7, 5, 9);
+ * let Names = Observable.of<string>('Foo', 'Bar', 'Beer');
+ * let Gender = Observable.of<string>('F', 'M', 'F');
```

Comment:
Can you lowercase these variable names? Also I think a lot of our other examples use Finnish Notation (i.e. `age$` `name$` `gender$`.

Finally, I think I'd like to exclude gender from this example and use something else. There are people who don't fit into the "M" or "F" pattern of gender, and in support of those folks, I think it would be best to stick to neutral material.